.circleci/config.yml

```diff
@@ -129,6 +129,20 @@ jobs:
           command: make citest fork=merge
       - store_test_results:
           path: tests/core/pyspec/test-reports
+  test-capella:
```

Comment:
this new CI job has to be added to the "workflows".

Reply:
got it!